core/src/epicli/data/common/defaults/configuration/opendistro-for-elasticsearch.yml

```diff
@@ -1,9 +1,20 @@
 kind: configuration/opendistro-for-elasticsearch
-title: OpenDistro for Elasticsearch Config
+title: Open Distro for Elasticsearch Config
 name: default
 specification:
   cluster_name: EpiphanyElastic
   clustered: True
+  admin_password: PASSWORD_TO_CHANGE
```

Comment:
The same suggestion as for `defaults/configuration/logging.yml`

Reply:
Comment the same as for logging.